src/ssl.c

```diff
@@ -11057,6 +11071,203 @@ int wolfSSL_use_certificate(WOLFSSL* ssl, WOLFSSL_X509* x509)
 
 #endif /* OPENSSL_EXTRA */
 
+#if defined(HAVE_RPK)
+
+static int isArrayUnique(const char* buf, size_t len)
+{
+    /* check the array is unique */
+    for (size_t i = 0; i < len -1; ++i) {
```

Comment:
move declaration of type to top of function or in the case of 'j' to top of outside for loop rather than declared inside the for loop